### DISCUSSION TITLE
Where are the logs saved?

### DISCUSSION THREAD

**Discussion by crimsonduelist** [dail8859/NotepadNext#601] [Q&A]
Where are the logs saved?

**Comment by dail8859** [comment]
I'm assuming you are talking about the debug log under the help menu? These don't get saved, they are only accessible within the application. Depending on the OS and how it is ran it may also print these out to the console.

**Reply by crimsonduelist** [reply]
> I'm assuming you are talking about the debug log under the help menu? These don't get saved, they are only accessible within the application. Depending on the OS and how it is ran it may also print these out to the console.

Well I wanted to open a issue about getting 
[     1.045] I: Primary instance already running. PID: 6557
[     1.146] I: Secondary instance closing...
and i cant find the pid actually running even pkilling it i ll  still get this (running it from console)
And since I dont know how to reproduce it I thought I could only provide the logs.
Is there any other kind of loggin being done besides those debug logs? How could I get them in console, is it documented somwhere so I could check it out?
I m on linux.

**Reply by dail8859** [reply]
The third-party library https://github.com/itay-grudev/SingleApplication is responsible for the primary/secondary instance handling so likely an issue with that libary. There have been reports of other issues with it as well. E.g. #541

Those debug logs are the only ones. Its however Qt handles the output. Not sure how Linux handles it, especially whe it comes to the various ways it can be installed. I know at least on Windows you can tell Qt to redirect the logging to the standard output.
